pkg/download/download.go

```diff
@@ -0,0 +1,17 @@
+package download
+
+import (
+	"context"
+	"io"
+
+	"github.com/gomods/athens/pkg/storage"
+)
+
+// Protocol is the download protocol defined by cmd/go
```

Comment:
> defined by cmd/go

do you mean that this is taken from `cmd/go`?

Reply:
Sorry the wording is definitely misleading. I meant they mirror the HTTP requests that cmd/go expects to use. I'll update the comment.